server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityGraphsHeader.js

```diff
@@ -33,11 +34,12 @@ type Props = {
   selectedMetrics: Array<string>,
   updateGraph: string => void
 };
+*/
 
 export default class ProjectActivityGraphsHeader extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  handleGraphChange = (option: { value: string }) => {
+  handleGraphChange = (option /*: { value: string } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)